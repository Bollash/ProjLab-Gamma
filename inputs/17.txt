Create Space
Add Asteroid Uran
Add Asteroid Iron
Add Settler 0
Add Settler 1
Add Settler 1
Mod Layer 0 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Countdown 0
Status All