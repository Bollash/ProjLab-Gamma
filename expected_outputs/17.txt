Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
Sikeres volt a telepeshozzáadás
Sikeres volt a telepeshozzáadás
Sikeres volt a telepeshozzáadás
Az adott aszteroida kérge a kapott értékre lett állítva.
A napközeli számláló eggyel csökkent.
A napközeli számláló eggyel csökkent.
Napközelbe került az aszteroida.
Az urán számlálója eggyel nőtt.
A napközeli számláló eggyel csökkent.
A napközeli számláló eggyel csökkent.
Napközelbe került az aszteroida.
Az urán számlálója eggyel nőtt.
A napközeli számláló eggyel csökkent.
A napközeli számláló eggyel csökkent.
Napközelbe került az aszteroida.
Az aszteroida felrobbant.
Space
2
10
10
False
1
2
Asteroid
0
3
3
3
Iron
0
2
0
1
Settler
0
0
0
0
Settler
1
0
0
0